charts/osm/values.yaml

```diff
@@ -169,6 +169,9 @@ OpenServiceMesh:
   # -- Deploy Jaeger during OSM installation
   deployJaeger: false
 
+  # -- Enable multicluster 
+  enableMulticluster: false
```

Comment:
I have already created a feature flag for this here : https://github.com/openservicemesh/osm/blob/4c390a5026874c5e35f52457dc9fb1c818375ea0/charts/osm/values.yaml#L227

Please could we use this